src/common/motion/position.ts

```diff
@@ -376,9 +376,9 @@ export class Position extends vscode.Position {
    * Gets the position one to the left of this position. Does not go up line
    * breaks.
    */
-  public getLeft(): Position {
+  public getLeft(count: number = 1): Position {
     if (!this.isLineBeginning()) {
```

Comment:
Can you update this check to ensure that we don't go past the beginning of the line?

Reply:
there's no check for getRight()